specification/loadtestservice/LoadTestService.Management/routes.tsp

```diff
@@ -0,0 +1,104 @@
+import "./models.tsp";
+import "./customOperation.tsp";
+import "@azure-tools/typespec-providerhub";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.ResourceManager;
+using OpenAPI;
+using Autorest;
+using Azure.ResourceManager.Foundations;
+
+namespace Microsoft.LoadTestService;
+
+interface Operations extends Azure.ResourceManager.Operations {}
+
+@@encodedName(LoadTests.createOrUpdate::parameters.resource,
+  "application/json",
+  "LoadTestResource"
```

Comment:
why do you want to rename the default `resource` to this? Since this a wrapping node, it doesn't matter for encoding. You can use @clientName (used by TSP native codegen) and/or `x-ms-client-name` (used by autorest codegen).

Reply:
This is already published and expected behaviour. We are expecting something like (LoadTestResource : LoadTestResource ) <-- ??? instead of (resource: LoadTestResource). As suggested by you in office hours, we are using encodedName to update json produced instead of using projectedNames.

Reply:
Have added client.tsp files with @@clientName usage.